### DISCUSSION TITLE
How do I let TeXStudio know where my TeXLive installation is?

### DISCUSSION THREAD

**Discussion by kleinbottle** [texstudio-org/texstudio#1725] [Q&A]
I am new to `TeXStudio` and on first firing up, am greeted by a warning that `LaTeX` has not been found.

I have a `TeXLive` installation on Manjaro Linux rooted at `/usr/local/texlive/2021/bin/x86_64-linux/` for the executable binaries. All my environment variables like `PATH` have been set and `mktexlsr` is up to date. 

1. Why does `TeXStudio` need help to find my installation?
2. How might I help `TeXStudio` to find the installation?

**Comment by sunderme** [comment]
you can see under help/check latex installation whta PATH setting txs sees.
When starting from GUI, probably .profile is used to determined path settings.
